Makefile

```diff
@@ -118,6 +118,16 @@ KPROMO_PKG := sigs.k8s.io/promo-tools/v3/cmd/kpromo
 CONVERSION_VERIFIER_BIN := conversion-verifier
 CONVERSION_VERIFIER := $(abspath $(TOOLS_BIN_DIR)/$(CONVERSION_VERIFIER_BIN))
 
+OPENAPI_GEN_VER := v0.0.0-20211115234752-e816edb12b65
```

Comment:
Looks like they don't have releases, but should we use the latest commit?

Reply:
We can, for now I did choose the same as we had on the poc branch (via go.mod), will update it 👍